gulpfile.js

```diff
@@ -12,42 +11,39 @@ requireDir('./config');
  */
 
 gulp.task('build', function(cb) {
-  runSequence('sass', 'js:concat', 'js:vendor', 'fractal:build', 'css:prefix-fractal', cb);
+  gulp.series('sass', 'js:concat', 'js:vendor', 'fractal:build', 'css:prefix-fractal', cb);
```

Comment:
Not sure you need this `cb` argument here. that was used as a callback from the `runSequence` plugin that we're no longer using. I would definitely test that though.